### PR TITLE
Integrate visits and messages apis for VAOS

### PR DESCRIPTION
**Pull request by jbalboni**
## Description
This integrates the past visits and messages apis on VAOS. The past visits endpoint works on staging right now, however messages does not. But since we catch errors from the messages call and don't fail the new request process, it doesn't hurt us to push it to staging.

## Testing done
Local and unit testing

## Acceptance criteria
- [ ] Past visits are checked, message POST is attempted

## Definition of done
- [ ] Events are logged appropriately
- [ ] Documentation has been updated, if applicable
- [ ] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)
- [ ] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs

### CODE REVIEW

**Comment by jbalboni** [comment]
Coverage is again low because of the mock data that we're not unit testing

**Review by narin — APPROVED** [review]
Could maybe use a unit test for `getUserMessage()` but looks good.

**Comment by jbalboni** [comment]
@narin Added one.

File: src/applications/vaos/api/index.js [hunk 1]
```diff
@@ -478,7 +497,7 @@ export function getPreferences() {
 
 export function updatePreferences(data) {
   let promise;
-  if (USE_MOCK_DATA || true) {
+  if (USE_MOCK_DATA) {
```
Comment (narin) [inline src/applications/vaos/api/index.js]: 👍🏼

**Review by narin — APPROVED** [review]

**Review by rianfowler — APPROVED** [review]

**Pull request merged**